llm_bench/python/utils/config_class.py

```diff
@@ -11,7 +11,7 @@
     OVLatentConsistencyModelPipeline,
     OVStableDiffusionXLPipeline
 )
-from utils.ov_model_classes import OVMPTModel, OVLDMSuperResolutionPipeline, OVChatGLMModel
+from llm_bench.python.utils.ov_model_classes import OVMPTModel, OVLDMSuperResolutionPipeline, OVChatGLMModel
```

Comment:
```suggestion
from .ov_model_classes import OVMPTModel, OVLDMSuperResolutionPipeline, OVChatGLMModel
```

Reply:
I wonder if it is possible to use the relative approach and make it working as well.

Reply:
it depends from which directory script used. If it runs from openvino.genai/llm_bench.python, then using relative path will not work (attempt to use relative import from root directory) as well as the current proposed solution (this approach does not permit to run from internal directory at all). Variant with relative import will be ok, if we made llm_bench installable package with entry point.

We need to clarify how it is generally used.
@peterchen-intel @wgzintel @skuros could you please share command how you run llm_bench.

For me, the most universal will be if utils will be renamed to llm_bench_utils to be more specific and avoid issues with relative directory usage, however it will requires more changes across different scripts.

Reply:
I'm using llm_bench together with [[sstrehlk/lm-evaluation-harness](https://github.com/sstrehlk/lm-evaluation-harness/blob/6d05dd6648bbf9d9bdbc7763fcf7bc92bef0f3f9/lm_eval/models/optimum_lm_genai.py#L149)](https://github.com/sstrehlk/lm-evaluation-harness/blob/6d05dd6648bbf9d9bdbc7763fcf7bc92bef0f3f9/lm_eval/models/optimum_lm_genai.py#L149) and there is a conflict with utils.py here.  The name llm_bench_utils seems to be good solution.

Reply:
In the performance benchmarking is it used in the following way:
`numactl -N 1 --membind=1 python3.10 benchmark.py -mc 1 -ic 128 -m <model> -d <device> -n 3 -r <model>_report.csv -rj <model>.json -pf <model>_prompt.jsonl`

And in the conversion:
`. venv/bin/activate && cd llm_bench/python && python3.10 convert.py --output_dir <model>  --compress_weights_backends openvino --precision FP16 --model_id <model>`

Reply:
I changed name of utils to llm_bench_utils according to @eaidova suggestion. The master is merged to  sstrehlk:update-utils-path branch. PR is ready for merging.